### PR TITLE
Update samples to beta3

### PR DESCRIPTION
**Pull request by friism**
I've finally gotten the CLA signed

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@friism__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla2.msopentech.com.
    </span>

TTYL, MSOTBOT;

**Comment by friism** [comment]
@msotclas I've gotten the CLA signed already.

**Comment by ahmetalpbalkan** [comment]
@friism can you please squash commits into one commit?

@eilon can we review this, blocks docker image release for beta3.

**Comment by friism** [comment]
@ahmetalpbalkan commits squashed

File: samples/ConsoleApp/project.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "aspnetcore50": {
 	"dependencies": {
                 "System.Console": "4.0.0-beta-*"
-            }
-}
+	}
```
Comment (Eilon) [inline samples/ConsoleApp/project.json]: Tabs :frowning: :panda_face:

**Comment by Eilon** [comment]
@anurse can you evaluate? I vaguely recall you mentioning in another PR about how we want to flow changes for samples (i.e. through dev or not).

**Comment by friism** [comment]
@Eilon tabs removed

**Comment by friism** [comment]
@Eilon @anurse I'm unsure if flowing changes through dev branch makes sense for this, since it's not a new feature but a bug-fix.

**Comment by Eilon** [comment]
@anurse can you take a look on Monday?

**Comment by Eilon** [comment]
Marking as CLA-ok... I manually checked the database.

**Comment by muratg** [comment]
Looks good to me. I'll let @anurse put the ship it label.

**Comment by ahmetalpbalkan** [comment]
@muratg @anurse @Eilon ping. this is blocking Docker image release.

File: samples/ConsoleApp/project.json [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
   },
   "frameworks": {
     "aspnet50": {},
     "aspnetcore50": {
-	"dependencies": {
-                "System.Console": "4.0.0-beta-*"
-            }
-}
+      "dependencies": {
+        "System.Console": "4.0.0-beta-*"
```
Comment (Praburaj) [inline samples/ConsoleApp/project.json]: This package version can also be fixed. Version will be `4.0.0-beta-22605`

**Pull request merged**